src/main/java/seedu/address/logic/commands/UpdateFoodIntakeCommand.java

```diff
@@ -0,0 +1,82 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.time.LocalDate;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.food.Food;
+import seedu.address.model.food.FoodIntake;
+
+/**
+ * Updates a particular FoodIntake item in the FoodIntakeList.
+ */
+public class UpdateFoodIntakeCommand extends Command {
+    public static final String COMMAND_WORD = "updatefoodintake";
```

Comment:
I feel the command should be rephrased to foodintakeupdate for consistency with the rest

Reply:
Good point! Will update this.

Reply:
Updated command name.